packages/e2e/README.md

```diff
@@ -0,0 +1,168 @@
+# End to end tests for Cardano JS SDK
+
+This directory contains end-to-end tests for cardano-js-sdk. Each directory under 'test' is a self-contained set of tests for a particular component/use case that exactly mimics how a user might expect the cardano-js-sdk to work, allowing high-fidelity reproductions of real-world issues.
+
+For this to work, we first install dependencies and build the sdk from the root via:
+
+```bash
+$ yarn cleanup
```

Comment:
How about 2 scripts:
- `reinstall` = `cleanup:nm` (stands for node_modules, already exists) + `install`
- `rebuild` = `cleanup:dist` + `build`

Reply:
Also a word I've used before for install & build is `setup`

Reply:
I like the second approach, because sometimes you want to rebuild but not necessarily want to reinstall all dependencies. But I think a single script to do a completely clean install will also be very useful. Which approach should we follow?

Reply:
done here https://github.com/input-output-hk/cardano-js-sdk/pull/305/commits/0452ae3fa479b2bde6697dadbd3766668a97f95e